mrmap/quality/views.py

```diff
@@ -21,6 +23,13 @@
 class ConformityCheckRunListView(SecuredListMixin, FilterView):
     model = ConformityCheckRun
     table_class = ConformityCheckRunTable
+    filterset_class = ConformityCheckRunFilterSet
+    queryset = ConformityCheckRun.objects.select_related("config", "service", "service__service_type",
```

Comment:
move queryset things like select_related stuff to a manager function called like 'for_table_view'